### PR TITLE
resource/alicloud_click_house_db_cluster: Added support for cluster version 23.8, added support for in-place cluster storage upgrade. resource/alicloud_click_house_account: Added support for creating super account.

### PR DESCRIPTION
**Pull request by itkillsnone**
resource/alicloud_click_house_db_cluster: Added support for cluster version 23.8, added support for in-place cluster storage upgrade.
resource/alicloud_click_house_account: Added support for creating super account.

### CODE REVIEW

File: alicloud/resource_alicloud_click_house_account.go [hunk 1]
```diff
@@ -49,11 +49,14 @@ func resourceAlicloudClickHouseAccount() *schema.Resource {
...
-			"status": {
-				Type:     schema.TypeString,
-				Computed: true,
-			},
 			"type": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				Default:      "Normal",
+				ValidateFunc: validation.StringInSlice([]string{"Normal", "Super"}, false),
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_click_house_account.go:57]: "validation. "去掉，另外不要设置默认值

File: alicloud/resource_alicloud_click_house_account.go [hunk 2]
```diff
@@ -75,7 +78,6 @@ func resourceAlicloudClickHouseAccount() *schema.Resource {
 			},
 			"total_databases": {
 				Type:     schema.TypeString,
-				Optional: true,
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_click_house_account.go:78]: 这个为啥移除了

File: alicloud/resource_alicloud_click_house_account.go [hunk 3]
```diff
@@ -85,7 +87,6 @@ func resourceAlicloudClickHouseAccount() *schema.Resource {
 			},
 			"total_dictionaries": {
 				Type:     schema.TypeString,
-				Optional: true,
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_click_house_account.go:88]: 为啥去掉了Optional

File: alicloud/resource_alicloud_click_house_account.go [hunk 4]
```diff
@@ -158,6 +163,9 @@ func resourceAlicloudClickHouseAccountRead(d *schema.ResourceData, meta interfac
 	d.Set("allow_dictionaries", convertArrayToString(authority["AllowDictionaries"], ","))
 	d.Set("total_databases", convertArrayToString(authority["TotalDatabases"], ","))
 	d.Set("total_dictionaries", convertArrayToString(authority["TotalDictionaries"], ","))
+	if err != nil {
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_click_house_account.go:166]: 这里的 err 不是上面已经判定过了吗

File: alicloud/resource_alicloud_click_house_account.go [hunk 5]
```diff
@@ -262,12 +270,6 @@ func resourceAlicloudClickHouseAccountUpdate(d *schema.ResourceData, meta interf
 		request["AllowDatabases"] = v
 	}
 
-	if d.HasChange("total_databases") {
-		update = true
-	}
-	if v, ok := d.GetOk("total_databases"); ok {
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_click_house_account.go:268]: 为啥移除了

File: alicloud/resource_alicloud_click_house_account.go (hunk 1 above)
Comment (itkillsnone) [reply alicloud/resource_alicloud_click_house_account.go:57]: 好的

File: alicloud/resource_alicloud_click_house_account.go (hunk 3 above)
Comment (itkillsnone) [reply alicloud/resource_alicloud_click_house_account.go:88]: total_dictionaries为Computed，由后端计算得到，作为入参无效

File: alicloud/resource_alicloud_click_house_account.go (hunk 5 above)
Comment (itkillsnone) [reply alicloud/resource_alicloud_click_house_account.go:268]: 不作为入参后，应当移除

File: alicloud/resource_alicloud_click_house_account.go (hunk 4 above)
Comment (itkillsnone) [reply alicloud/resource_alicloud_click_house_account.go:166]: 这个err是调用DescribeClickHouseAccountAuthority后的err，之前并未判定

File: alicloud/resource_alicloud_click_house_account.go (hunk 5 above)
Comment (ChenHanZhang) [reply alicloud/resource_alicloud_click_house_account.go:268]: image
API 参数上还有，不应该移除。

Comment (itkillsnone) [reply alicloud/resource_alicloud_click_house_account.go:268]: 那我先将其标为deprecated，未来API移除后TF再移除。

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request closed without merging**